packages/manager/apps/dedicated/client/app/ip/translations/Messages_fr_FR.json

```diff
@@ -755,5 +755,25 @@
   "ip_repricing_banner_list_item_2": "A partir du 1er décembre 2022, pour vos adresses existantes (commandées avant le 6 octobre 2022).",
   "ip_repricing_banner_link": "Plus d'informations",
   "ip_unused_banner_text": "Vous avez des adresses IP non utilisées. Les adresses IPv4 étant une ressource qui se raréfie, si vous n'en avez pas besoin, nous vous invitons à les relâcher.",
+  "ip_region_label": "Région : ",
+  "ip_region_eu-west-par": "Europe (France - Paris)",
+  "ip_region_eu-west-gra": "Europe (France - Gravelines)",
+  "ip_region_eu-west-rbx": "Europe (France - Roubaix)",
+  "ip_region_eu-west-sbg": "Europe (France - Strasbourg)",
+  "ip_region_eu-west-lim": "Europe (Allemagne - Limburg)",
+  "ip_region_eu-central-waw": "Europe (Pologne - Varsovie)",
+  "ip_region_eu-west-eri": "Europe (Royaume-Uni - Erith)",
+  "ip_region_us-east-vin": "Amérique du Nord (États-Unis - Est - Vinthill)",
+  "ip_region_us-west-hil": "Amérique du Nord (États-Unis - Ouest - Hillsboro)",
+  "ip_region_ca-east-bhs": "Amérique du Nord (Canada - Est - Beauharnois)",
+  "ip_region_ap-southeast-sgp": "Asie-Pacifique (Singapour - Singapour)",
+  "ip_region_ap-southeast-syd": "Asie-Pacifique (Australie - Sydney)",
+  "ip_region_eu-west-rbx-snc": "Europe (France - Roubaix) (SNC)",
+  "ip_region_eu-west-sbg-snc": "Europe (France - Strasbourg) (SNC)",
+  "ip_region_ca-east-tor": "Amérique du Nord (Canada - Est - Toronto)",
+  "ip_region_ap-south-mum": "Asie-Pacifique (Inde - Mumbai)",
+  "ip_region_labeu-west-1-preprod": "LABEU (France - Croix) (PREPROD)",
+  "ip_region_labeu-west-1-dev-1": "LABEU (France - Croix) (DEV-1)",
+  "ip_region_labeu-west-1-dev-2": "LABEU (France - Croix) (DEV-2)",
```

Comment:
**question:** I think the list is not complete as I dont see lot of other DCs like Australia, Singapore, US ad so on.

Reply:
I relied on the associated Confluence page and from what I see everything is here